libs/permission-checker/src/TokenPermission.php

```diff
@@ -6,6 +6,6 @@
 
 enum TokenPermission: string
 {
-    case CAN_RUN_JOBS = 'canRunJobs';
```

Comment:
![image](https://github.com/keboola/platform-libraries/assets/1726727/2457197b-5dec-40e4-9621-bbeed22fbd69)

takhle by to mělo být správně